File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02613/p02613_num8_parsed.cpp
#include <iostream>
#include <map>
#include <string>

int main() {
    int N;
    std::cin >> N;
    
    std::map<std::string, int> verdicts;
    
    for (int i = 0; i < N; i++) {
        std::string verdict;
        std::cin >> verdict;
        verdicts[verdict]++;
    }
    
    std::cout << "AC x " << verdicts["AC"] << std::endl;
    std::cout << "WA x " << verdicts["WA"] << std::endl;
    std::cout << "TLE x " << verdicts["TLE"] << std::endl;
    std::cout << "RE x " << verdicts["RE"] << std::endl;
    
    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02613/p02613_num8_parsed.py
def count_verdicts(N, verdicts):
    count_AC = count_WA = count_TLE = count_RE = 0
    
    for verdict in verdicts:
        if verdict == "AC":
            count_AC += 1
        elif verdict == "WA":
            count_WA += 1
        elif verdict == "TLE":
            count_TLE += 1
        elif verdict == "RE":
            count_RE += 1
    
    print(f"AC x {count_AC}")
    print(f"WA x {count_WA}")
    print(f"TLE x {count_TLE}")
    print(f"RE x {count_RE}")

# Get input
N = int(input())
verdicts = []
for _ in range(N):
    verdicts.append(input())

# Call the function
count_verdicts(N, verdicts)